frontend/packages/dev-console/src/components/catalog/providers/useBuilderImageSamples.ts

```diff
@@ -27,10 +30,12 @@ const normalizeBuilderImages = (
     const provider = annotations?.[ANNOTATIONS.providerDisplayName] ?? '';
     const creationTimestamp = imageStream.metadata?.creationTimestamp;
     const href = `/samples/ns/${activeNamespace}/${name}/${imageStreamNS}`;
+    const createLabel = t('devconsole~Create Application');
+    const type = 'Sample';
 
     const item: CatalogItem = {
-      uid,
-      type: 'Sample',
+      uid: `${type}-${uid}`,
```

Comment:
Why is this needed? Isn't uid already unique?

Reply:
nope, even dev catalog has builder images and they share same ID